e2e/cypress/integration/enterprise/cloud/onboarding/cloud_sysadmin_onboarding_spec.js

```diff
@@ -8,6 +8,7 @@
 // ***************************************************************
 
 // Group: @enterprise @onboarding
+// Skip:  @electron @chrome @firefox
```

Comment:
Temporarily skip Cloud feature since trial license don't have it. Will investigate further on how to deal with Cloud features.